_data/navigation.yml

```diff
@@ -0,0 +1,24 @@
+---
+- title: 'Work'
+  link: '/#work'
+  type: local
+
+- title: 'About'
+  link: '/#about'
+  type: local
+
+- title: 'Join VAC'
```

Comment:
VAC should be Vac everywhere. Alternatevely, "vac.dev" where it makes sense. We never use VAC with capital letters